src/test/java/com/hotels/beeju/ThriftHiveMetaStoreCoreTest.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (C) 2015-2019 Expedia, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.beeju;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertTrue;
+
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.junit.Test;
+
+import com.hotels.beeju.core.BeejuCore;
+import com.hotels.beeju.core.ThriftHiveMetaStoreCore;
+
+public class ThriftHiveMetaStoreCoreTest {
+
+  private final BeejuCore core = new BeejuCore();
+  private final ThriftHiveMetaStoreCore thriftHiveMetaStoreCore = new ThriftHiveMetaStoreCore(core);
+
+  @Test
+  public void initalisedThriftCore() {
+    assertNotNull(thriftHiveMetaStoreCore.getThriftServer());
```

Comment:
I don't think the ExecutionService should be allowed to leak out of the class. That method `getThriftServer()` should be private.